astropy/time/formats.py

```diff
@@ -281,10 +274,11 @@ def set_jds(self, val1, val2):
 
         # Possible enhancement: use np.unique to only compute start, stop
         # for unique values of iy_start.
-        jd1_start, jd2_start = erfa_time.dtf_jd(
-            self.scale.upper().encode('utf8'), iy_start, imon, iday, ihr, imin, isec)
-        jd1_end, jd2_end = erfa_time.dtf_jd(
-            self.scale.upper().encode('utf8'), iy_start + 1, imon, iday, ihr, imin, isec)
+        scale = self.scale.upper().encode('utf8')
```

Comment:
Is there a reason to encode this as `utf8`?  Presumably the ERFA C routines are explicitly only able to accommodate ASCII encoded values.  I understand that in practice it is moot, but I was just wondering if there was an actual driver.